src/com/lilithsthrone/game/character/body/types/EyeType.java

```diff
@@ -17,50 +20,254 @@
+			Race.ALLIGATOR_MORPH,
+			1,
+			EyeShape.ROUND,
+			EyeShape.VERTICAL,
+			"reptile-like",
+			(Boolean isPc) -> {
+				return (isPc ? EyeTransformationText.PC_START + EyeTransformationText.PC_TO_ALLIGATOR
+						: EyeTransformationText.NPC_START + EyeTransformationText.NPC_TO_ALLIGATOR) + EyeTransformationText.FINISH;
+				}
+		),
 
-	HORSE_MORPH(BodyCoveringType.EYE_HORSE_MORPH, Race.HORSE_MORPH, 1, EyeShape.ROUND, EyeShape.HORIZONTAL),
+	HORSE_MORPH(
+			BodyCoveringType.EYE_HORSE_MORPH,
+			Race.HORSE_MORPH,
+			1,
+			EyeShape.ROUND,
+			EyeShape.HORIZONTAL,
+			"horse-like",
+			(Boolean isPc) -> {
+				return (isPc ? EyeTransformationText.PC_START + EyeTransformationText.PC_TO_HORSE
+						: EyeTransformationText.NPC_START + EyeTransformationText.NPC_TO_HORSE) + EyeTransformationText.FINISH;
+				}
+		),
 
-	REINDEER_MORPH(BodyCoveringType.EYE_REINDEER_MORPH, Race.REINDEER_MORPH, 1, EyeShape.ROUND, EyeShape.HORIZONTAL),
+	REINDEER_MORPH(
+			BodyCoveringType.EYE_REINDEER_MORPH,
+			Race.REINDEER_MORPH,
+			1,
+			EyeShape.ROUND,
+			EyeShape.HORIZONTAL,
+			"reindeer-like",
+			(Boolean isPc) -> {
+				return (isPc ? EyeTransformationText.PC_START + EyeTransformationText.PC_TO_REINDEER
+						: EyeTransformationText.NPC_START + EyeTransformationText.NPC_TO_REINDEER) + EyeTransformationText.FINISH;
+				}
+		),
 
-	HARPY(BodyCoveringType.EYE_HARPY, Race.HARPY, 1, EyeShape.ROUND, EyeShape.ROUND);
+	HARPY(
+			BodyCoveringType.EYE_HARPY,
+			Race.HARPY,
+			1,
+			EyeShape.ROUND,
+			EyeShape.ROUND,
+			"bird-like",
+			(Boolean isPc) -> {
+				return (isPc ? EyeTransformationText.PC_START + EyeTransformationText.PC_TO_HARPY
+						: EyeTransformationText.NPC_START + EyeTransformationText.NPC_TO_HARPY) + EyeTransformationText.FINISH;
+				}
+		);
 
-	
 	private BodyCoveringType coveringType;
 	private Race race;
 	private int eyePairs;
-	private EyeShape irisShape, pupilShape;
+	private EyeShape irisShape;
+	private EyeShape pupilShape;
+	private String descriptor;
+	private Function<Boolean, String> transformationText;
 
-	private EyeType(BodyCoveringType coveringType, Race race, int eyePairs, EyeShape irisShape, EyeShape pupilShape) {
+	private EyeType(
+			BodyCoveringType coveringType,
+			Race race,
+			int eyePairs,
+			EyeShape irisShape,
+			EyeShape pupilShape,
+			String descriptor, 
+			Function<Boolean, String> transformationText) {
 		this.coveringType = coveringType;
 		this.race = race;
 		this.eyePairs = eyePairs;
 		this.irisShape = irisShape;
 		this.pupilShape = pupilShape;
+		this.descriptor = descriptor;
+		this.transformationText = transformationText;
+	}
+	
+	public Race getRace() {
+		return this.race;
+	}
+	
+	public int getEyePairs() {
+		return this.eyePairs;
+	}
+	
+	public EyeShape getIrisShape() {
+		return this.irisShape;
+	}
+	
+	public EyeShape getPupilShape() {
+		return this.pupilShape;
+	}
+	
+	public Function<Boolean, String> getTransformationText() {
+		return this.transformationText;
```

Comment:
To build off AlacoGit's comment, I personally think this is okay Java, however, it might be rather unexpected. I would suggest moving the apply inside the getter unless there is a good reason to do it elsewise:
```java
public String getTransformationText(boolean isPc) {
    return this.transformationText.apply(isPc);
}
```

Reply:
I liked your suggestion about moving the apply.

As far as Java goes, I think it feels unexpected because a lot of the functional programming elements in Java 8 are still uncommon to see. I've only seen this sort of thing used a couple times, and this is only my second implementation, but I think it makes the calling code easier to read and it isn't to hard to figure out what it is doing if you have to look at the methods.